helpers_test.go

```diff
@@ -0,0 +1,53 @@
+package kv
```

Comment:
Ah, whoops. You're right, thanks! I can move these to a non `_test.go` file in a  separate test helpers package to make it obvious and import it directly

Reply:
Actually, after a bit of research this might not be true. Forgive me, since I'm new to Go. It seems that a file ending in `_test.go` indicates that the file should be packaged up when running `go test`. I don't think it necessarily dictates that the file _must_ contain tests. Creating a file that doesn't end in `_test.go` would mean that it would be packaged into the binary which I don't think we want. Thoughts?

Reply:
All that you're saying is true.  Nonetheless, we typically put helpers in non-_test.go files, because otherwise they can't be imported by other packages.

The Go linker is pretty smart, I don't think it'll include the helper code in the prod binary unless prod code invokes it.

Reply:
Of course if the helpers aren't public anyway, it's up to you.

Reply:
That makes sense. I would like to follow our general practices so I will move things. Thanks for the feedback everyone.

Reply:
So now looking at this more, having a separate package will result in cyclic imports as the `getBackend` function requires the `VersionedKVFactory`. I didn't like that the `getBackend` function was defined in a `path_data_test.go` and instead wanted it in a separate file. Though it's all the same test package so maybe I'm just complicating things and might just revert all that and go back to way things were prior to this PR.